foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/platform/database/SQLServerPlatform.java

```diff
@@ -813,11 +813,11 @@ public void printSQLSelectStatement(DatabaseCall call, ExpressionSQLPrinter prin
             return;
         }
         
-        // OFFSET + FETCH NEXT requires ORDER BY, so add an ordering if there are none
-        // this SQL will satisfy the query parser without actually changing the ordering of the rows
+        // OFFSET + FETCH NEXT requires ORDER BY
         List<Expression> orderBy = statement.getOrderByExpressions();
         if (orderBy.isEmpty()) {
-            orderBy.add(statement.getBuilder().literal("ROW_NUMBER() OVER (ORDER BY (SELECT null))"));
```

Comment:
I think, that this removal it should lead into error at line No 835.

Reply:
How so? It returns with the change in 820.

Reply:
Sorry it was my mistake I missed `return;` part.